src/DataViews/CallstackDataView.cpp

```diff
@@ -138,6 +138,42 @@ std::vector<std::string> CallstackDataView::GetContextMenu(
   return menu;
 }
 
+std::vector<std::vector<std::string>> CallstackDataView::GetContextMenuWithGrouping(
+    int clicked_index, const std::vector<int>& selected_indices) {
```

Comment:
Can `selected_indices` be empty here? (size() == 0)

Reply:
I'm not a QT expert so I might be wrong here.

According to `OrbitTreeView::ShowContextMenu`, I think the `selected_indices` won't be empty when `GetContextMenuWithGrouping` is called.

`GetContextMenuWithGrouping` is called when the index returned by `indexAt(...)` is valid. And in this case, I guess, the selection list returned by `selectedIndexes()` should not be empty.

Reply:
Okay. That makes sense to me. Its not clear when reading only this code location. Maybe a CHECK and/or comment is a good idea here.